kytos-amlight-topo-init.sh

```diff
@@ -0,0 +1,172 @@
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:13:3  '{"mtu": 9000, "port_name": "Interface-13:3"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:13:5  '{"mtu": 9000, "port_name": "Interface-13:5"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:14:7  '{"mtu": 9000, "port_name": "Interface-14:7"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:14:8  '{"mtu": 9000, "port_name": "Interface-14:8"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:15:4  '{"mtu": 9000, "port_name": "Interface-15:4"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:15:6  '{"mtu": 9000, "port_name": "Interface-15:6"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:15:7  '{"mtu": 9000, "port_name": "Interface-15:7"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:16:5  '{"mtu": 9000, "port_name": "Interface-16:5"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:16:6  '{"mtu": 9000, "port_name": "Interface-16:6"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:17:10 '{"mtu": 9000, "port_name": "Interface-17:10"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:17:9  '{"mtu": 9000, "port_name": "Interface-17:9"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:18:11 '{"mtu": 9000, "port_name": "Interface-18:11"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:18:13 '{"mtu": 9000, "port_name": "Interface-18:13"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:18:14 '{"mtu": 9000, "port_name": "Interface-18:14"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:18:16 '{"mtu": 9000, "port_name": "Interface-18:16"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:19:12 '{"mtu": 9000, "port_name": "Interface-19:12"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:19:13 '{"mtu": 9000, "port_name": "Interface-19:13"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:19:15 '{"mtu": 9000, "port_name": "Interface-19:15"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:20:16 '{"mtu": 9000, "port_name": "Interface-20:16"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:20:17 '{"mtu": 9000, "port_name": "Interface-20:17"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:21:14 '{"mtu": 9000, "port_name": "Interface-21:14"}'
+set_interface_enable_lldp_metadata 00:00:00:00:00:00:00:21:15 '{"mtu": 9000, "port_name": "Interface-21:15"}'
+
+
+#############################
+# Setup links
+#############################
+# curl -s http://127.0.0.1:8181/api/kytos/topology/v3/links | jq -r '.links[] | .id + " \"" +(.endpoint_a.name|tostring) + "--" + (.endpoint_b.name|tostring) + "\""'
+wait_on links 17
+set_link_enable_metadata 1e7e531ae50f419b1d35d311a66351d997ea567348b691d8dccdc639e1f8340a '{"availability": 100.0, "latency": 1, "packet_loss": 0.0, "bandwidth": 100, "residual_bandwidth": 100.0, "link_name": "Ampath4-eth11--Ampath1-eth11"}'
```

Comment:
@italovalcy from what I recall, we ended up removing `latency` and `residual_bandwidth` in favor of `delay` and `utilization` (and derive the residual from 100 - utilization) https://github.com/kytos-ng/kytos-ng.github.io/commit/6f69774d3d46129b5a7e2e8b77a34b2c66282e18, could you double check this or if we need to review the reserved metadata again?

Reply:
@italovalcy I've pushed this commit https://github.com/amlight/kytos-end-to-end-tests/pull/132/commits/026f07a6954cd73d9b652fc07e92be30adefc33a to help out to get this PR merged and addressed this point, let me know if you're OK with it.